README.md

```diff
@@ -1,2 +1,15 @@
 # u04-webshop
-U04 Webshop
+
+This is a simple single-page webshop. 
+
+Productdata is fetched from [https://fakestoreapi.com]
+
+Webpage deployed on GitHub Pages: [https://chas-henrik.github.io/u04-webshop/]
+
+## Built with *HTML, CSS and Javascript*. 
+There is a main indexfile for each language and for javascript there are two extra files designated to contain specific functions which are then imported to the main file. 
```

Comment:
We don't have an `index.html` for each language, only one for English as far as I know?
We don't support multiple langages as far as I know, or maybe I misunderstand the text?

Reply:
I mean a file called index for each "tech-language", a bit of a shortcut since i know html and css aren't technically languages. It is a bit unclear now that you mention it. I will rewrite that